prototype/core/geo-json/src/main/java/org/eclipse/sensinact/gateway/geojson/GeoJsonObject.java

```diff
@@ -61,4 +63,44 @@ protected GeoJsonObject(GeoJsonType type) {
     public Map<String, Object> foreignMembers = new HashMap<>();
 
     public List<Double> bbox;
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(type, bbox);
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (obj instanceof GeoJsonObject) {
```

Comment:
Add a comment indicating that this is safe because it's only intended to be used as part of an equals in a concrete subclass.